entities/src/main/java/org/odk/collect/entities/javarosa/intance/LocalEntitiesInstanceAdapter.kt

```diff
@@ -0,0 +1,86 @@
+package org.odk.collect.entities.javarosa.intance
+
+import org.javarosa.core.model.data.StringData
+import org.javarosa.core.model.instance.TreeElement
+import org.odk.collect.entities.javarosa.parse.EntityItemElement
+import org.odk.collect.entities.storage.EntitiesRepository
+import org.odk.collect.entities.storage.Entity
+
+class LocalEntitiesInstanceAdapter(private val entitiesRepository: EntitiesRepository) {
```

Comment:
I'm thinking we'll eventually pull out an interface (`InstanceDataAdapter` for example) from this to use for normal secondary instances as well. The entity `InstanceProvider` and `FilterStrategy` are both actually generic enough to support that with minimal changes and could be pulled out into another package, but that felt too far for the moment.